### DISCUSSION TITLE
Is it possible to generate images using only model.pt file without converting to ckpt file?

### DISCUSSION THREAD

**Discussion by healthyfat** [kohya-ss/sd-scripts#118] [Q&A]
At the moment to check the result of Lora training I convert created model.pt file into model.ckpt and then run "gen_img_diffusers" script to generate sample images. Is there a way to do it without the need to convert model?

**Comment by tommyjohn81** [comment]
automatic1111 now has built in lora support so you just need to put your Lora model.pt file in the models/lora folder and make sure the Lora extension is enabled.  Then in your prompt you can call on your lora model by using:

<lora:modelname:1>  where 1 is the weight you want to use

**Reply by healthyfat** [reply]
Thanks, but only interested in using Kohya.

**Comment by kohya-ss** [comment]
Sorry for late reply. ``gen_img_diffusers.py`` supports LoRA model. Three options ``--network_*`` are required like this:

```
python gen_img_diffusers.py --ckpt model.ckpt --n_iter 1 --scale 11 --steps 40 --outdir ../txt2img  --xformers 
--W 512 --H 512 --bf16 --sampler k_euler_a 
--network_module networks.lora --network_weights trained_lora.safetensors  --network_mul 1.0 
--max_embeddings_multiples 3 --clip_skip 2 --batch_size 1 --images_per_prompt 1 --interactive
```

You can also specify multiple LoRAs like this:
```
--network_module networks.lora networks.lora 
--network_weights trained_lora.safetensors1 trained_lora2.safetensors 
--network_mul 0.5 0.7 
```

Three options must have the same number of arguments.

**Reply by healthyfat** [reply]
How I supposed to create this file? 

trained_lora.safetensors

When I do LORA training it results in:

trained_lora.pt

Is there no way to use model generated with LORA training without need to convert?

**Reply by healthyfat** [reply]
Ok. So it works now. Thanks.

I did upgrade and after that it creates *.safetensors file automatically. I guess it was just matter of choosing right output format. 

Does it mean that *.pt file are not supported?
